tests/utils/push_proxy/test_service.py

```diff
@@ -0,0 +1,94 @@
+from datetime import date
+from unittest.mock import call
+
+import pytest
+
+from utils.push_proxy.service import join_s3_path, load_stage_to_table
+
+
+@pytest.mark.parametrize(
+    "prefix,parts",
+    [
+        pytest.param("/path1/path2/path3", ["extra1", "extra2"], id="leading /"),
```

Comment:
Is it guaranteed that `prefix` always start with `/`? If not maybe we can add a test case for that setup.